Python/READme.md.txt

```diff
@@ -0,0 +1,2 @@
+Python basic programs
+ 
```

Comment:
You have to write a short description. Just writing about it in one line doesn't suffice. Also please use any one convention when naming the readme. It should either be `README.md` or `readme.txt`, not both at a time. Github recommends README.md because .md can be natively viewed and styled in [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)

Reply:
@adijdhv Do let me know if you are facing any issues we are here to help you. You can also use the WhatsApp group to get your queries resolved